src/main/main.ts

```diff
@@ -106,6 +107,23 @@ export function setupTitleBarClickMac() {
   });
 }
 
+function isNativeThemeSource(
+  val: unknown,
+): val is typeof nativeTheme.themeSource {
```

Comment:
Under what conditions would this return `false`? I could _maybe_ see the case for checking that it's 'dark', 'light', or 'system' but i don't see how it wouldn't be a string 🤔